deepchem/rl/a3c.py

```diff
@@ -237,7 +237,7 @@ def __init__(self, a3c, index):
       self.train_op = a3c._graph._get_tf('Optimizer').apply_gradients(
           grads_and_vars)
       self.update_local_variables = tf.group(
-          * [tf.assign(v1, v2) for v1, v2 in zip(local_vars, global_vars)])
+          *[tf.assign(v1, v2) for v1, v2 in zip(local_vars, global_vars)])
```

Comment:
yapf is complaining about this change.  It's one of those super irritating cases where yapf produces a different result on some computers than others, so if you run it locally, it produces formatting that travis rejects as wrong.